scripts/saml_script/saml_get_token.py

```diff
@@ -0,0 +1,100 @@
+# sslverification = 'apexiam-cert.pem'
+# read config json to load the idp and saml ur's
+config_file_path = "config.json"
+try:
+    with open(config_file_path, encoding="utf8") as config_file:
+        config_data = json.load(config_file)
+    oktaloginendpoint = config_data["okta"]["login_endpoint"]
+    oktasamlendpoint = config_data["okta"]["saml_endpoint"]
+    diurl = config_data["Exchange_URL"]
+except FileNotFoundError:
+    print("Config file not found.")
+    exit(0)
+except KeyError:
+    print("key not found in the config file. Please update the url in config file.")
+    exit(0)
+    
+def save_config(file_path, config_data):
+            with open(file_path, 'w') as config_file:
+                json.dump(config_data, config_file, indent=4)
+#provide chocie to user to choose
+def let_user_pick(options):
+    print("Please choose:")
+
+    for idx, element in enumerate(options):
+        print("{}) {}".format(idx + 1, element))
+
+    i = input("Enter number: ")
+    print()
+    try:
+        if 0 < int(i) <= len(options):
+            return int(i) - 1
+    except:
+        pass
+    return None
+options = ["okta",'exit']
+res = 0 #let_user_pick(options)
+if options[res] in config_data:
+    idp_config = config_data[options[res]]
+    if "okta"  in options[res].lower():
+       print(f"The below are example parameters vaules for okta saml_endpoint and login_endpoint:")
+       print()
+       print(f"login_endpoint:" +" https://foo.okta.com/home/foo_premiertestapp_1/abhgjk789/abcde456D")
+       print()
+       print(f"saml_endpoint:"+"  https://foo.okta.com/home/foo_testapp_1/abfgj00996/sso/saml")
+       print()
+       print(f"Current configuration for the url's:")
+       print()
+       print(f"IDP URL: {idp_config['login_endpoint']}")
+       print()
+       print(f"SAML URL: {idp_config['saml_endpoint']}")
+       print()
+    edit_choice = "no" #input("Do you want to edit this IDP's configuration? (yes/no): ").strip()
+    if edit_choice.lower() == 'yes':
+        if "okta"  in options[res].lower():
+            print()
+            idp_config['login_endpoint'] = input("Enter New IDP URL: ").strip()
+            print()
+            edit_choice = input("Do you want to edit this SAML configuration? (yes/no): ").strip()
+            print()
+            if edit_choice.lower() == 'yes':
+                idp_config['saml_endpoint'] = input("Enter New SAML URL: ").strip()
+            else:
+                print("none")
+        save_config(config_file_path, config_data)
+        print()
+        print("Configuration updated successfully, re-run the script after url update.")
+        sys.exit(0)
+    else:
+            def handler():
+                """main handler"""
+                if "okta" in options[res].lower():
+                    #subprocess.run(["python", "saml_get_token_okta_idp.py"], input=idpentryurl, text=True, stdout=subprocess.PIPE)
```

Comment:
Remove the commented out code, and add a comment that this the `saml_get_token_idp.py` is the one that needs to be updated to either `win or lin`

Reply:
I added - Copy either win or lin file to saml_get_token_okta_idp.py as per the platform in README. Instead of changing code.